### DISCUSSION TITLE
Link verification useragent

### DISCUSSION THREAD

**Discussion by fanonwue** [mastodon/mastodon#23154] [Q&A]
Hello everyone,

I run my own link-shortener that I wrote myself, and want to have the green checkmark next to my "main" website on my mastodon profile. It just redirects to a carrd.co link, so I was thinking about checking the useragent in my link-shortener code to render a small HTML file with the verification link instead of redirecting if it's coming from a mastodon link verification service.

I am no expert in Ruby and certainly not familiar with the Mastodon codebase, can someone either tell me how the useragent looks like or give me a hint where I can find it?

An alternative would be to simply use a GET parameter, but that looks a bit ugly in my opinion.

**Comment by mjankowski** [comment]
It's going to be something like:

> Mastodon/4.6.0-alpha.1 (http.rb/5.3.1; +http://mastodon.test/)

Full string value will vary from server to server, and depending on versions, etc.
